### DISCUSSION TITLE
Relative asset paths

### DISCUSSION THREAD

**Discussion by woutermont** [digita-ai/semcom#78] [Brainstorm]
Do we have any idea how semcom will/should behave when the client requests a component whose HTML contains a relative path to an asset (e.g. images)? Do we require all assets (even large ones) to be bundled in a single JS file?

**Comment by lem-onade** [comment]
https://stackoverflow.com/a/33603356/14407010

**Comment by woutermont** [comment]
Try [`import.meta.url`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta). Otherwise, add an `origin` event that is fired by the SDK on registration, passing the component's URL, and on which the component code should listen if it needs this.
